dist/localforage.js

```diff
@@ -1828,6 +1829,10 @@ requireModule('promise/polyfill').polyfill();
         global.webSQLStorage = mod.exports;
     }
 })(this, function (exports, module) {
+    'use strict';
+
+    var _systemImportTransformerGlobalIdentifier = typeof window !== 'undefined' ? window : self;
```

Comment:
Oh shoot. I didn't mean to do that it must have done the build with an old version. I'll patch that up ASAP. Thanks. 



—
Sent from Mailbox

On Sun, Aug 2, 2015 at 8:43 AM, Thodoris Greasidis
<notifications@github.com> wrote:

>> @@ -1828,6 +1829,10 @@ requireModule('promise/polyfill').polyfill();
>>          global.webSQLStorage = mod.exports;
>>      }
>>  })(this, function (exports, module) {
>> +    'use strict';
>> +
>> +    var _systemImportTransformerGlobalIdentifier = typeof window !== 'undefined' ? window : self;
> Sorry for the naming :disappointed: but this variable name is generated by babel-plugin-system-import-transformer@1.1.4 using the babel-provided `generateUidIdentifier` method to avoid conflicts.
> The variable itself is used in the code transformations of `System.import` calls instead of just using `var global = window`.
> This
> * is required for web/service workers &
> * allows for better minification, since it appears only once in the outemost scope of each transformed file that uses `System.import`. (In older versions this would appear once per `System.import` transformation.
> It seems that [this was reverted](https://github.com/mozilla/localForage/blob/cf6f1d11218e3453463f243e684d996c7a2eaf76/dist/localforage.js#L1568-L1581) by ceab4ad15695e148e3a49ff2a3881264e3eeb841.
> Can you please confirm that the latest versions of the npm package was used before this commit?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mozilla/localForage/pull/406/files#r36040512